e2e/explorer/api/src/app/etherscan/account.ts

```diff
@@ -91,3 +92,43 @@ export function balanceHistory(params: GetParams<BalanceHistory>) {
   if (!balance) return '0';
   return balance.toString();
 }
+
+export function tokenTx(params: GetParams<TokenTx>) {
+  const {address, contractAddress, startblock = 0, endblock, page, sort='asc'} = params;
+  if (!address || !contractAddress) throw new Error('Error! Missing address or contract address');
+  const transferID = id('Transfer(address,address,uint256)');
+
+  const sorting = {
+    asc: (a: TransactionReceipt, b: TransactionReceipt) => a.blockNumber - b.blockNumber,
+    desc: (a: TransactionReceipt, b: TransactionReceipt) => b.blockNumber - a.blockNumber
+  };  
+
+  // get the address transactions logs
+  const logs = store.addresses[address].transactions
+    .map(log => store.logs[log])
+    .flat()
+    .filter(log => {
+      if (log.address !== address) return false;
```

Comment:
you can already get the logs per address with : 
```suggestion
  const logs = store.logs[address].filter(log => {
```
Like that you don't even have to verify if the log.address is the same as the address in the query